agent.md

```diff
@@ -1,46 +1,76 @@
-  --set kubecostProductConfigs.clusterName=<custom cluster name> \
+  --set prometheus.server.global.external_labels.cluster_id=<CLUSTER_NAME> \
+  --set kubecostProductConfigs.clusterName=<CLUSTER_NAME> \
+  --set prometheus.nodeExporter.enabled=true \
+  --set prometheus.serviceAccounts.nodeExporter.create=true \
+  --set networkCosts.enabled=true \
   --set-file agentKey="kubecost-agent.key"
 ```
 This step will install:
-* kubecost-agent Deployment and service
-* prometheus-server Deployment and service 
-* node-exporter DaemonSet
+* kubecost-agent deployment and service
+* prometheus-server deployment and service
+* node-exporter daemonSet (set to false if you already have node-exporter running)
+* network-costs daemonSet (optional, collects additional metrics used for egress cost visibility) [learn more](https://guide.kubecost.com/hc/en-us/articles/4407595973527)
+
+## Additional Clusters
 
-Optionally, the `--set networkCosts.enabled=true` can be used during the helm install to include the `kubecost-network-costs` DaemonSet. [Learn more](https://docs.kubecost.com/network-allocation.html)
+For multi-cluster setups, all additional cluster installs would use the following install command:
 
-For multicluster setups, all additional cluster installs would use the following install command:
 ```bash
-helm install kubecost kubecost/cost-analyzer \
-  --namespace kubecost \
+helm upgrade --install kubecost kubecost/cost-analyzer \
+  --namespace kubecost-agent --create-namespace\
   --values=https://raw.githubusercontent.com/kubecost/cost-analyzer-helm-chart/develop/cost-analyzer/values-agent.yaml \
-  --set kubecostToken="bWJvbHQzNUBnbWFpbC5jb20=xm343yadf98" \
-  --set prometheus.server.global.external_labels.cluster_id=<unique cluster identifier> \
-  --set kubecostProductConfigs.clusterName=<custom cluster name> \
+  --set prometheus.server.global.external_labels.cluster_id=<CLUSTER_NAME> \
+  --set kubecostProductConfigs.clusterName=<CLUSTER_NAME> \
   --set kubecostMetrics.exporter.exportClusterInfo=false \
   --set kubecostMetrics.exporter.exportClusterCache=false \
+  --set prometheus.nodeExporter.enabled=true \
+  --set prometheus.serviceAccounts.nodeExporter.create=true \
+  --set networkCosts.enabled=true \
   --set-file agentKey="kubecost-agent.key"
 ```
 
-3. Confirm with Kubecost team on successful deployment, which will then provide an endpoint `http://<your-organization>.kubecost.io` which can be used to access all exported data. 
+3. Confirm with Kubecost team on successful deployment, which will then provide access to the hosted UI: `https://<your-organization>.kubecost.io` which can be used to access all exported data.
+
+> Note: Metrics are shipped every 2 hours, a delay is expected when viewing on the UI.
+
+
+## Troubleshooting Agent Deployments
+
+It is common to have prometheus errors when the kubecost-agent pod starts. They should not continue after the kubecost-prometheus-server pod is ready.
```

Comment:
It might be helpful to show what we expect on the output here to verify prom is running properly along with the agent pod

Reply:
i.e. https://cert-manager.io/docs/installation/verify/#manual-verification

Reply:
agreed. done.